RC 82: Manuscript Cases and other changes (#87)
highfidelity/hifi-docs-sphinx: docs/source/create/3d-models/pbr-materials-guide.rst

1. If you set transparency with a texture, the transparency (alpha) should be in the material’s albedo texture, as a PNG file with transparency and not as a separate transparency texture.
2. High Fidelity’s renderer can draw two different kinds of transparency: "alpha" (255 graduates steps of transparency, no shading on surface, casts no shadows,) and "mask" (binary transparency, full shading of opaque surface, whole surface casts shadow.)
3. To determine whether a texture is treated as a mask or as alpha, the engine looks for alpha values between 2% and 98%. An easy way to create a mask texture is to save your image as a PNG-8 since it only supports binary transparency, while PNG-24 supports a range of transparency levels.

> 4. Currently, we support using a second UV set with the following texture channels only - Light map and Ambient Occlusion. 

Reviewer: I want to remove anything that suggests we'll "fix" or change it in the future. So...

```suggestion
4. To add a second UV set, use either the Light Map or Ambient Occlusion texture channels. ```

Author: I've changed it a bit to keep the terminology of using the second UV set.

> 4. Currently, we support using a second UV set with the following texture channels only - Light map and Ambient Occlusion. 

Author: ```suggestion
4. We support using a second UV set with the following texture channels only - Light map and Ambient Occlusion. 
```
